tests/unit/api/main.py

```diff
@@ -2376,3 +2384,80 @@ def test_node_power_cycle_force(self):
         new_node('node-99')
         api.project_connect_node('anvil-nextgen', 'node-99')
         api.node_power_cycle('node-99', True)
+
+
+class TestShowNetworkingAction:
+    """Various tests for the show networking action api"""
+
+    def common(self):
```

Comment:
Better to inherit from unittest.TestCase and just call this setUp (and then not explicitly call it below).